config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,9 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+      provider:               "AWS",
```

Comment:
Unnecessary spacing detected.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.